spec/models/user_spec.rb

```diff
@@ -822,9 +826,17 @@
 
     describe '#reinstate!' do
       before do
-        user.suspended_at = Time.zone.now
+        user.suspend!
         user.reinstated_at = nil
       end
+
+      it 'destroys SuspendedEmail records for each email address' do
+        email_address = user.email_addresses.last
+        expect { user.reinstate! }.to change {
+                                        SuspendedEmail.find_with_email(email_address.email)
+                                      }.from(email_address).to(nil)
```

Comment:
I think rubocop might get mad? I'd also use `do...end` for multiline blocks:

```suggestion
        expect { user.reinstate! }.
          to(change { SuspendedEmail.find_with_email(email_address.email) }.
            from(email_address).to(nil))
```

Reply:
yes autocorrect :D